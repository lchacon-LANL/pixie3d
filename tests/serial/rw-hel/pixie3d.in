 &datin
    ilevel   = 0

    test     = t

    nxd      = 32
    nyd      = 32
    nzd      = 1

    xmax     = 1.
    ymax     = 0.
    zmax     = 1.

    numerical_grid = f

    solenoidal = t
    nc_eom_f   = t
    nc_eom_v   = t
    solve_rho  = f

    tau_w = 1d2
    r_v   = 1.5

    limit_dt_flow_cfl = f
    
    bcs      = 'spt','rsw','per','per','per','per'

    eta      = 1d0  !Vacuum
    nu       = 1d0  !Vacuum
    dd       = 0d-3
    chi      = 0d-2
    
    aa_eta   = 0.
    bb_eta   = 0.
    
    equil    = 'rwhel'

    coords   = 'hel'

    gparams  = 1d0,-1d0,4d0,0d0,0d0,0d0,0d0

    gamma    = 1d0

    dt      = 1d-2
    numtime = 10
    ndstep  = 1

    pvx      = 0d-4
    odd      = f
    random   = f

    cnfactor = 0.
    sm_flag  = 2

    restart  = f
    timecorr = f

    postprocess = t

    source   = f
    chk_src  = f

    pc_type  = 's1'
    pc_tol   = 1d-1
    pc_iter  = 1
    pc_debug = f
    pc_debug_it = 0

    mg_vcyc  = 1
    mg_ores  = 0
    mg_oprol = 2
    mg_zebra_relax = f
    sm_iter  = 4

    iguess   = 0

    maxksp   = 20
    maxitgm  = 20
    tolgm    = 8d-1
    rtol     = 1d-3
    method   = 1
    mf_eps   = 1d-6
    vol_wgt  = t

    check_grid = f

!    gp1 = .false.,0d0,0.1
 /

 &graphics
    sel_diag  = 5,6,7,12,15,17,18,19,20,22,23
    sel_graph = 1,2,3,5,6,7,9,10,11
    dplot = 1d-4
    prof_conf = 1,'r',1,1,1
    cont_conf = 3,'x','y',1,1,1
    hdf_plot = f
 /

