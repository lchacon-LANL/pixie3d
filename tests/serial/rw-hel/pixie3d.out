
  A-solve GMRES its= 16

  Grid mesh:................   32x   32x    1
  Number of grid levels:....    4,    4,    1
  Preconditioning method:... SI PC w/o flow
  Timestep adaption......... F
  External source........... F
  Numerical grid............ F
  Scalar advection scheme... ZIP      
  Velocity advection scheme. ZIP      
  Anisotropic transport sch. 4th-7pt  
  TS scheme................. BDF2     

    itime      time     dt     #CFL   vx_max   vy_max   vz_max    GMRES Newton CN factor  GMRES-DC
 Computing RW BCs...
         1      0.00  1.0E-03 3.4E+02  2.2E-02  1.7E-02  8.4E-03    13     5     0.000       10
 Computing RW BCs...
         2      0.01  9.0E-03 3.0E+03  1.9E-02  1.8E-02  8.2E-03    16     5     0.000       10
 Computing RW BCs...
         3      0.02  1.0E-02 3.4E+03  2.2E-02  1.7E-02  7.7E-03    23     6     0.000       10
 Computing RW BCs...
         4      0.03  1.0E-02 3.4E+03  2.3E-02  1.7E-02  7.0E-03    21     5     0.000        8
 Computing RW BCs...
         5      0.04  1.0E-02 3.4E+03  2.3E-02  1.6E-02  6.4E-03    22     5     0.000        7
 Computing RW BCs...
         6      0.05  1.0E-02 3.4E+03  2.3E-02  1.5E-02  5.7E-03    28     6     0.000        7
 Computing RW BCs...
         7      0.06  1.0E-02 3.4E+03  2.3E-02  1.4E-02  5.1E-03    27     6     0.000        7
 Computing RW BCs...
         8      0.07  1.0E-02 3.4E+03  2.3E-02  1.3E-02  4.5E-03    28     6     0.000        7
 Computing RW BCs...
         9      0.08  1.0E-02 3.4E+03  2.3E-02  1.2E-02  4.0E-03    22     5     0.000        7
 Computing RW BCs...
        10      0.09  1.0E-02 3.4E+03  2.3E-02  1.1E-02  3.5E-03    22     5     0.000        7

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt
   10       5.4       22.2        4.1
